modules/ocf_kubernetes/files/abac.jsonl

```diff
@@ -0,0 +1 @@
+{"apiVersion": "abac.authorization.kubernetes.io/v1beta1", "kind": "Policy", "spec": {"group":"ocfdeploy",     "namespace": "*",              "resource": "*",         "apiGroup": "*"                   }}
```

Comment:
Could you remove the extra whitespace in this file (or even better, format it more nicely across lines if that's possible)

Reply:
JSONL is supposed to be a format similar to CSV, and each entry is supposed to be one line. I'll remove the whitespace though because nothing needs to line up yet.